themes/haystack/layouts/_default/tutorials.html

```diff
@@ -1,120 +1,177 @@
 {{/* Tutorials page */}}
 {{ define "main" }}
-  <div class="tutorials container container-padded">
-    <div class="inner inner-top">
-      {{/* Heading & subtitle */}}
-      <section class="tutorials-heading">
-        <h1>{{ .Params.title }}</h1>
-        <p>{{ .Params.subtitle | markdownify }}</p>
-      </section>
-
-      <section class="tutorials-buttons">
-        {{/* Filter buttons */}}
-        <div class="button-container">
-          {{ if eq .Params.categoryFilters.enabled true }}
-            {{ range .Params.categoryFilters.list }}
-              <button class="btn filter-btn enabled category-{{ lower . }}">
-                {{ . }}
+  <div class="tutorials">
+    {{/* Hero */}}
+    <div class="tutorials-hero">
+      <div class="container container-padded">
+        <section class="tutorials-heading">
+          <h1>{{ .Params.title }}</h1>
+          <p>{{ .Params.subtitle | markdownify }}</p>
+        </section>
+      </div>
+    </div>
+
+    <div class="tutorials-content">
+      <div class="container container-padded">
+        <div class="inner inner-top">
+          {{/* Section title */}}
+          <h2>
+            {{ len (where .Site.Pages "Layout" "tutorial") }} tutorials for
```

Comment:
When I select the beginner level in the filter, this text stays the same, but it should be "4 tutorials for beginner level" if possible